src/collective/exportimport/configure.zcml

```diff
@@ -161,7 +161,7 @@
       />
 
   <browser:page
-      zcml:condition="installed plone.app.multilingual"
+      zcml:condition="installed plone.app.multilingual.interfaces.ITranslationManager"
```

Comment:
@erral Unfortunately I think this only works with a module or package and won't work with an object inside a module namespace. The `installed` check is implemented here using `__import__`: https://github.com/zopefoundation/zope.configuration/blob/master/src/zope/configuration/xmlconfig.py#L367

and this is what happened when I tried to import ITranslationManager that way:
```
>>> __import__('plone.app.multilingual.interfaces.ITranslationManager')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'plone.app.multilingual.interfaces.ITranslationManager'; 'plone.app.multilingual.interfaces' is not a package
```

Reply:
Maybe you can add a conditional inside the view code instead?

Reply:
And how did it worked for me? :/

Reply:
It worked because I didn't have it enabled in my buildout :roll_eyes: